openedx/core/djangoapps/course_apps/management/commands/cache_course_app_status.py

```diff
@@ -5,18 +5,39 @@
 from textwrap import dedent
 
 from django.core.management import BaseCommand
+from opaque_keys.edx.keys import CourseKey
 
-from openedx.core.djangoapps.course_apps.tasks import cache_all_course_apps_status
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from openedx.core.djangoapps.course_apps.tasks import cache_all_course_apps_status, update_course_apps_status
 
 
 class Command(BaseCommand):
     """
     Command to cache status of course apps to the database to speed up queries.
 
+    If no arguments are provided, update cache for all courses.
     This can be run as a one-off command. If new plugins are installed or existing plugins are uninstalled, the
     status of those will be cached on first access so re-running this isn't necessary.
+
+
+    Examples:
+
+        ./manage.py lms cache_course_app_status
+        ./manage.py lms cache_course_app_status course-v1:edX+DemoX+Demo_Course course-v1:edX+CS101+T22021
```

Comment:
Do we have an idea of how many courses we will have to run this command for? 
It might become ugly/tedious adding course name after course name to the command line if the number of courses is significantly higher. Just wanted to checkin on this!